docker-compose.yml

```diff
@@ -22,7 +28,7 @@ services:
     build:
       context: .
       dockerfile: Dockerfile.mysql
-    container_name: my-sql
+    container_name: mysql
```

Comment:
The `container_name` directive is used here. Be aware that hardcoding container names can lead to conflicts if multiple instances of the same service are run. Consider whether this is necessary for your setup.